Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -116,15 +116,6 @@ function Write-TaskSpecificTelemetry
     Write-Telemetry "$codeKey" "3B5693D4-5777-4FEE-862A-BD2B7A374C68"
 }
 
-function ThrowError
```

Comment:
updated for windows file copy . For azure file copy we can check with PM and update it